src/async-wrap.cc

```diff
@@ -181,6 +183,37 @@ static void Initialize(Local<Object> target,
 }
 
 
+void AsyncWrap::DestroyIdsCb(uv_idle_t* handle) {
+  Environment* env = Environment::from_destroy_ids_idle_handle(handle);
+  // None of the V8 calls done outside the HandleScope leak a handle. If this
+  // changes in the future then the SealHandleScope wrapping the uv_run()
+  // will catch this can cause the process to abort.
+  v8::Context::Scope context_scope(env->context());
```

Comment:
Can you at least add a SealHandleScope?  That said, a HandleScop isn't _that_ expensive to create and it's more robust.  Debugging a crash or a leak six months from now because something changed would be highly annoying.